test/sqllogictest/explain/optimized_plan_as_text.slt

```diff
@@ -753,3 +753,26 @@ Source materialize.public.r
   filter=((#0) IS NOT NULL AND (#2) IS NOT NULL AND (#3) IS NOT NULL AND (#4) IS NOT NULL AND (#6) IS NOT NULL AND (#8) IS NOT NULL AND (#9) IS NOT NULL AND (#11) IS NOT NULL AND (#12) IS NOT NULL AND (#13) IS NOT NULL AND (#15) IS NOT NULL AND (#16) IS NOT NULL)
 
 EOF
+
+
+# Regression test for #19148: EXPLAIN path should refle
+statement ok
+DROP TABLE IF EXISTS t CASCADE;
+
+statement ok
+CREATE TABLE t(a TIMESTAMP);
+
+statement ok
+CREATE DEFAULT INDEX ON t;
+
+query T multiline
+EXPLAIN SELECT * FROM t WHERE a < mz_now();
+----
+Explained Query (fast path):
+  Filter (timestamp_to_mz_timestamp(#0) < 1683652808481)
```

Comment:
I find this output a bit odd because we have actually executed part of this query, and are not explaining it. I assume the underlying problem is we can't keep this as `mz_now()`?

Reply:
> I find this output a bit odd

Me too!

> I assume the underlying problem is we can't keep this as `mz_now()`?

Yes. More precisely the `create_fast_path_plan` that we call to get the output of this test expects to see a `SafeMfp` which is only possible if we inline the value of `mz_now()`.

Reply:
Urgh, it's even worse - this test will always produce a different output on each run...

Reply:
I might be able to do a super ugly hack and just replace whatever the current TS is with `mz_now()` in the emitted fast path explain output.

Reply:
You can also use testdrive with a regex replace to support changing timestamps.

Reply:
I find this format fine. At least in my mental model, the explain output reflects what is executed in the compute dataflow (module some parts like `Finishing`) and the dataflow does not invoke `mz_now()`. It's also in line with other instances where the optimizer evaluates expressions during optimization, e.g.:
```sql
materialize=> explain select max(i) from (values (1), (2), (3)) as t(i);
        Optimized Plan
------------------------------
 Explained Query (fast path):+
   Constant                  +
     - (3)                   +
```

But I appreciate that my mental model doesn't necessary match that of our users.

Reply:
> Urgh, it's even worse - this test will always produce a different output on each run...

I changed the test to run only `statement ok` (it is enough to prevent the regression from the resolved issue).